tools/install_demo_configuration.bat

```diff
@@ -315,7 +315,7 @@ echo plugins.security.enable_snapshot_restore_privilege: true >> "%OPENSEARCH_CO
 echo plugins.security.check_snapshot_restore_write_privileges: true >> "%OPENSEARCH_CONF_FILE%"
 echo plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"] >> "%OPENSEARCH_CONF_FILE%"
 echo plugins.security.system_indices.enabled: true >> "%OPENSEARCH_CONF_FILE%"
-echo plugins.security.system_indices.indices: [".plugins-ml-config", ".plugins-ml-connector", ".plugins-ml-model-group", ".plugins-ml-model", ".plugins-ml-task", ".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".ql-datasources", ".opendistro-asynchronous-search-response*", ".replication-metadata-store", ".opensearch-knn-models", ".geospatial-ip2geo-data*"] >> "%OPENSEARCH_CONF_FILE%"
+echo plugins.security.system_indices.indices: [".plugins-ml-config", ".plugins-ml-connector", ".plugins-ml-model-group", ".plugins-ml-model", ".plugins-ml-task", ".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".ql-datasources", ".opendistro-asynchronous-search-response*", ".replication-metadata-store", ".opensearch-knn-models", ".geospatial-ip2geo-data*", ".opendistro-job-scheduler-lock"] >> "%OPENSEARCH_CONF_FILE%"
```

Comment:
Is it possible to rename this to opensearch, its been a long time since the project was opendistro

Reply:
@peternied is there general migration guidance on renaming system indices? The security index is `.opendistro_security`

Reply:
While I agree with the name change, I'm concerned how this may affect backwards compatibility/upgrades.

Reply:
> I'm concerned how this may affect backwards compatibility/upgrades.

Isn't the transition from non-system to system index going to force these same concerns?  If so - I'd like use to take the chance to rename.

Reply:
Not necessarily, there's no direct user interaction with the `.opendistro-job-scheduler-lock` index so converting this into a system index just adds guardrails so a user does not accidentally delete it. But regarding the index name change there is clear [documentation](https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#indices) against renaming indices ([reference issue](https://github.com/opensearch-project/opensearch-plugins/issues/17#issuecomment-847204447)).

Reply:
I see the same justification was used to decide against renaming the security system index : https://github.com/opensearch-project/security/issues/1822